### PR TITLE
add unit tests for extract.py file

### PR DESCRIPTION
**Pull request by caitlinturnidge**
- Added some unit tests for the extract script, all are passing

### CODE REVIEW

File: extract.py [hunk 1]
```diff
@@ -7,17 +7,6 @@
 NO_OF_PLANTS = 51
```
Comment (haydenjack) [inline extract.py]: Think you may have to make sure they are the same

Comment (caitlinturnidge) [reply extract.py]: as in the function I deleted? David has given approval

Comment (haydenjack) [reply extract.py]: Misunderstood it sorry, will approve now

**Review by haydenjack — DISMISSED** [review]
Looks good

**Comment by caitlinturnidge** [comment]
Just need it approving again please

**Review by haydenjack — APPROVED** [review]
👍 👍 👍 👍 👍 👍

**Pull request merged**